documentation/docs/60-appendix/60-glossary.md

```diff
@@ -34,9 +34,9 @@ In SvelteKit, client-side routing will be used by default, but you can skip it w
 
 ## SPA
 
-A single-page app (SPA) is an application in which all requests to the server load a single HTML file which then does client-side rendering of the requested contents based on the requested URL. All navigation is handled on the client-side in a process called client-side routing with per-page contents being updated and common layout elements remaining largely unchanged. SPAs do not provide SSR, which has the shortcoming described above. However, some applications are not greatly impacted by these shortcomings such as a complex business application behind a login where SEO would not be important and it is known that users will be accessing the application from a consistent computing environment.
+A single-page app (SPA) is an application in which all requests to the server load a single HTML file which then does client-side rendering of the requested contents based on the requested URL. All navigation is handled on the client-side in a process called client-side routing with per-page contents being updated and common layout elements remaining largely unchanged. SPAs are highly discouraged as they have much worse performance characteristics and harm SEO.
```

Comment:
I don't think we should change this section, it's missing all the nuance and "when is it fine" now.

Reply:
The sentence "SPAs do not provide SSR, which has the shortcoming described above" doesn't make sense as there are not shortcomings described above. I think it's referring to the SSR section which describes the advantages of SSR and is below, so we'll have to tweak this section at least a little. I've added back some language about when SPAs are fine though